tests/test_bookmarks.py

```diff
@@ -408,16 +392,26 @@ def bookmarks_test(self):
 
                         if stream == 'payments':
                             self.assertPaymentsEqual(updated_record, sync_record)
+                        elif stream == 'inventories':
+                            self.assertInventoriesEqual(updated_record, sync_record)
                         else:
                             self.assertDictEqual(updated_record, sync_record)
 
-    def assertPaymentsEqual(self, created_record, sync_record):
-        self.assertEqual(frozenset(created_record.keys()), frozenset(sync_record.keys()))
-        created_record_copy = deepcopy(created_record)
+    def assertInventoriesEqual(self, expected_record, sync_record):
+        self.assertEqual(frozenset(expected_record.keys()), frozenset(sync_record.keys()))
```

Comment:
If we could add a brief doc-string to these assertion methods explaining that square is inconsistent with ___ so we account for that by relaxing our assertions I think that would be good.

Reply:
Sure.